meilisearch-release-changelogs/v0.28.0.md

```diff
@@ -0,0 +1,233 @@
+# v0.28.0 release changelogs
+
+The main goal of this new release is to stabilize the Meilisearch API 🚀
+
+This release introduces a lot of breaking changes (especially regarding names), and we are sorry for this. The purpose is to guarantee what we care about the most—providing the best search engine experience—by stabilizing the Meilisearch API. There should not be any breaking changes to the API in the future.
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), the official documentation is not updated yet, but will be ready for the official v0.28.0 release. The work-in-progress documentation is available [here]().
```

Comment:
```suggestion
```

Reply:
So you can remove it directly from github before the release 🤓

Reply:
I will not remove it from this PR, so that I can copy/paste in my next release changelog
However I will not put it in the v0.28.0 changelog on Monday haha